### PR TITLE
[NUXE-50] Add landing page and getting started to docs

### PR DESCRIPTION
**Pull request by bh247484**
#### Screens

screencap

#### What does this PR do?

1. Adds a new landing page and getting started page to the Playbook docs/site.
2. Refreshes the nav with links to the getting started page and existing pages in Playbook (kits, tokens, utilities).
3. Conditionally renders page specific version of the sidebar and the nav based on the route.
4. Cleans and simplifies the `application.html.erb` layout, creates a separate layout for all `/kits` routes.
5. Adds a footer partial to the `application.html.erb` layout.
6. Rescope dark mode cookie to only affect kit pages.

#### Breaking Changes

No breaking changes.  All routes/views accounted for.

#### Runway Ticket URL

[Runway Story](https://nitro.powerhrg.com/runway/backlog_items/NUXE-50)

#### How to test this

Test Milano deploy to ensure everything behaves as expected and all routes link appropriately.

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **URGENCY** Please select a release milestone
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*

### CODE REVIEW

File: app/pb_kits/playbook/packs/site_styles/_getting_started.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@import "../../tokens/colors";
+@import "../../tokens/spacing";
+
+.getting_started {
+  display: flex;
+  justify-content: center;
+  padding: 32px 40px;
```
Comment (jasoncypret) [inline app/pb_kits/playbook/packs/site_styles/_getting_started.scss:7]:
```suggestion
  padding: $space_lg $space_xl;
```
Why not use our tokens?

File: app/pb_kits/playbook/packs/site_styles/_site-style.scss [hunk 2]
```diff
@@ -25,6 +25,22 @@ body {
...
     background: tint(#0a0527, 10);
     }
+    .mobile-popover-nav {
+      display: none;
+    }
+    @include break_at(breakpoint("lg")) {
+      .full-size-nav {
+        display: none;
+      }
+      padding: 20px 30px;
```
Comment (jasoncypret) [inline app/pb_kits/playbook/packs/site_styles/_site-style.scss:35]: Can you use our tokens?

```suggestion
      padding: $space_md $space_lg;
```

**Review by jasoncypret — CHANGES_REQUESTED** [review]

File: app/pb_kits/playbook/packs/site_styles/_getting_started.scss (hunk 1 above)
Comment (bh247484) [reply app/pb_kits/playbook/packs/site_styles/_getting_started.scss:7]: Missed this one, can definitely use spacing here.

File: app/pb_kits/playbook/packs/site_styles/_site-style.scss (hunk 2 above)
Comment (bh247484) [reply app/pb_kits/playbook/packs/site_styles/_site-style.scss:35]: Nav spacing is really precise doesn't work well with our spacing variables.  I'll use a differential here though (`$space_lg - 2`).

**Review by jasoncypret — APPROVED** [review]

**Pull request merged**